Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.8.5-alpine
```

Comment:
Can you use the image from quay?

Reply:
The one in quay.io is in 3.7. Since I tested the changes against this newer python version, I think it's fine to just use that newer version. I added the 3.8.5 image in our quay.io repo for base images.